addons/storyshots/storyshots-puppeteer/package.json

```diff
@@ -34,14 +34,14 @@
   },
   "dependencies": {
     "@axe-core/puppeteer": "^4.2.0",
-    "@storybook/csf": "0.0.2--canary.4566f4d.1",
+    "@storybook/csf": "0.0.2--canary.0899bb7.0",
     "@storybook/node-logger": "7.0.0-alpha.12",
     "@types/jest-image-snapshot": "^4.1.3",
     "core-js": "^3.8.2",
     "jest-image-snapshot": "^4.3.0"
   },
   "devDependencies": {
-    "@storybook/csf": "0.0.2--canary.4566f4d.1",
+    "@storybook/csf": "0.0.2--canary.0899bb7.0",
```

Comment:
Seems superfluous

Reply:
How do you mean? 

<img width="459" alt="image" src="https://user-images.githubusercontent.com/132554/178170144-41c5a49b-977b-46d3-abb7-8dcc462faf31.png">

Reply:
I meant that it's listed in both `dependencies` and `devDependencies`.